internal/backend/db/dbgorm/dbgorm.go

```diff
@@ -428,11 +450,20 @@ func delete[T any](db *gorm.DB, ctx context.Context, where string, args ...any)
 	return nil
 }
 
-func (db *gormdb) client(debug bool) *sql.DB {
-	q := db.db
+func (db *gormdb) client(debug bool) (r, w *sql.DB) {
```

Comment:
It's wrong for this function to panic instead of return an error, so I'd prefer you fix this, but it's not critical since your PR only extends the return type.

Reply:
done